http/headers/Idempotency-Key.json

```
@@ -0,0 +1,45 @@
+{
+  "http": {
+    "headers": {
+      "Idempotency-Key": {
+        "__compat": {
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": "135",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "network.http.idempotencyKey.enabled",
+                  "value_to_set": "true"
+                }
+              ],
+              "impl_url": "https://bugzil.la/1830022",
+              "notes": "Automatically adds a unique 64-bit key for POST and PATCH when needed, if not set in JavaScript."
```

Currently only POST looking at the implementation URL

```suggestion
              "notes": "Automatically adds a unique 64-bit key for POST requests when needed, if not set in JavaScript."
```